celobox/passwd.py

```diff
@@ -31,7 +34,9 @@ def new_session(self):
 
     def load_data(self, domain):
         try:
-            return json.load(open('manifests/%s.json' % domain))
+            return json.load(open(
+                resource_filename(__name__, os.path.join('manifests', '{}.json'.format(domain)))
```

Comment:
Ah I see – I did a search for `load_data()` and only looked at the first result. All good.